package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
@Toilal I know, but I don't think we should that either. Just imagine users having a lot of huge images and us reading all that in.